<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="PlotWidget" name="graphWidget" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_continue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>386</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Continue time loop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="box_pltgrid">
          <property name="text">
           <string>Plot model grid</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="widget_input">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Input variable name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="box_datatype">
          <item>
           <property name="text">
            <string>double</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>int</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_getval">
          <property name="text">
           <string>Get value</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
